src/config.js

```diff
@@ -0,0 +1,3 @@
+export const config = {
+  token: process.env.TOKEN,
+}
```

Comment:
`process.env.XXX` is how you read in your `.env` variables. 

The `.env` file is pulled in at the root of the project, which in this case is `src/index.js` with this line:

```js
import 'dotenv/config'
```

And then you can use it anywhere in your project. You can also add the `.env` variables directly to your machine, but that's better left for containers or prod servers. For hobby projects I often stick with just `.env` files anyway.

Reply:
Also generally you want a flag for dev or prod. This flag is usually named `NODE_ENV`. But you don't _need_ that yet. It's just a good way to have code run differently in dev vs prod. 

It's not an enforced standard, but it's a standard that everyone uses. Here's an article on it. 

https://nodejs.dev/learn/nodejs-the-difference-between-development-and-production